v21.1/foreign-key.md

```diff
@@ -26,7 +26,6 @@ To read more about how foreign keys work, see our [What is a Foreign Key? (With
 **Foreign Key Columns**
 
 - Foreign key columns must use their referenced column's [type](data-types.html).
-- A foreign key column cannot be a [computed column](computed-columns.html).
```

Comment:
This should read: "A foreign key column cannot be a [virtual computed column](...)" (see my second example above). You might want to clarify that a foreign key column **can** be a stored computed column, though.